package-lock.json

```diff
@@ -0,0 +1,6019 @@
+{
```

Comment:
I think this file is untracked on my codespace instance? Do we need this here.

Reply:
Good call!